.github/scripts/trymerge.py

```diff
@@ -972,16 +974,15 @@ def handle_exception(e: Exception, msg: str = "Merge failed") -> None:
         gh_post_comment(org, project, args.pr_num, "Cross-repo ghstack merges are not supported", dry_run=args.dry_run)
         return
 
-    if args.on_green or args.on_mandatory:
-        try:
-            merge_on_green(args.pr_num, repo, dry_run=args.dry_run, mandatory_only=args.on_mandatory)
-        except Exception as e:
-            handle_exception(e)
-    else:
-        try:
-            pr.merge_into(repo, dry_run=args.dry_run, force=args.force, comment_id=args.comment_id)
-        except Exception as e:
-            handle_exception(e)
+    try:
+        merge(args.pr_num, repo,
+              dry_run=args.dry_run,
+              force=args.force,
+              comment_id=args.comment_id,
+              mandatory_only=args.mandatory_only)
```

Comment:
There are no `mandatory_only` options in the args anymore, so this would always be false, isn't it? If you are deleting it from the options, perhaps you shall either delete it or keep the option (even if it can not be invoked via the bot)